Lab-3.ipynb

```diff
@@ -0,0 +1,728 @@
+    "\n",
+    "We can double check this answer using a different statistical PDF, the Rayleigh distribution; a Rayleigh with $\\sigma = 1$ is the same as a Chi distribution for k=2 (2 degrees of freedom, for a 2D normal vector distribution).\n",
+    "\n",
+    "https://en.wikipedia.org/wiki/Rayleigh_distribution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "5-Sigma detection; Probability = 3.7266531720536733e-06\n",
+      "Distance for detection = 4.4803\"\n"
+     ]
+    }
+   ],
+   "source": [
+    "# double check: a chi with k=2 deg. of freedom is the same as a Rayleigh with sigma=1\n",
+    "p2dRay = stats.rayleigh.cdf(5)\n",
+    "print(\"5-Sigma detection; Probability = \" + str(1-p2dRay))\n",
+    "\n",
+    "distRay = stats.norm.ppf(p2dRay)\n",
+    "print(\"Distance for detection = \" + str(\"{0:.4f}\".format(distRay)) + \"\\\"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "True\n",
+      "True\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(p2d == p2dRay)\n",
+    "print(dist == distRay)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This confirms the first calculation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 3: Cosmic Ray Backgrounds "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In a moon-sized patch on the sky, we know we detect cosmic rays at a rate of 1 ray per minute. Observing the moon for 8 hours a night for 15 days, we see a total of 6800 cosmic rays from this observational period. If a cosmic ray has an energy of 1 trillion eV or above, the moon will block the cosmic ray and the path of sky appears to be a shadow, or blank of cosmic rays, due to absorption. We want to know the significance behind a moon shadow detection. In other words, we want to determine the statistical significance of high energy cosmic rays above $1*10^{12}$ eV occurring within the moon-sized patch of sky we observe in the 15-day time period. Knowing this, we can ask the following statistical question:\n",
+    "\n",
+    "For a signal-free discrete background distribution, what is the probability that the background produces a cosmic ray detection with an energy equal to or higher than $1*10^{12}$ eV, which will not appear in the observational data (missing from the background)?"
```

Comment:
i believe our questions are different, however the one you state does make sense in the context given